src/astronomer/flask_appbuilder/security.py

```diff
@@ -305,7 +306,7 @@ def has_access(self, action_name, resource_name, **kwargs) -> bool:
 
 Airflow23CompatibilityMixin = object
 # Only define this if we're using an old version of Airflow
-if AIRFLOW_VERSION_TUPLE < (2, 3):
+if AIRFLOW_VERSION < Version("2.3.0"):
```

Comment:
Doesn't this have problems with the `dev` flag in our version

```suggestion
if AIRFLOW_VERSION.release < (2,3)```

Reply:
Cool, take a look now.